UI/window-importer.cpp

```diff
@@ -560,42 +547,30 @@ void OBSImporter::importCollections()
 		if (selected == Qt::Unchecked)
 			continue;
 
-		QString path = optionsModel->index(i, ImporterColumn::Path)
-				       .data(Qt::DisplayRole)
-				       .value<QString>();
-		QString name = optionsModel->index(i, ImporterColumn::Name)
-				       .data(Qt::DisplayRole)
-				       .value<QString>();
-
-		std::string pathStr = path.toStdString();
-		std::string nameStr = name.toStdString();
+		std::string pathStr =
+			optionsModel->index(i, ImporterColumn::Path)
+				.data(Qt::DisplayRole)
+				.value<QString>()
+				.toStdString();
+		std::string nameStr =
+			optionsModel->index(i, ImporterColumn::Name)
+				.data(Qt::DisplayRole)
+				.value<QString>()
+				.toStdString();
```

Comment:
Just for future reference, I'm not sure if I'd call this a refactor, doesn't really have any functional difference. I feel like it'd debatable whether it'd be considered cleaner.

Reply:
I used the word "refactor" in the commit message and PR title because simply saying "allow special characters in scene collection names in importer" seemed to deceptively gloss over the fact that I had moved a function to a completely different file location and removed code in favor of using a completely different code path.  Where before the importer used its own process for determining safe filenames and filename conflicts, now it uses a code path already implemented in OBS.  I felt that was fair play for using "refactor" to describe the overall changes, but would have used "change" or "modify" if that word was preferable.

As for this specific chunk of code, I removed these intermediate variables because they were not reused, and I judged that they served no purpose.  All the same, I would have been fine reverting or changing this chunk before merging if it was a problem.